content/en/events/guides/migrating_to_new_events_features.md

```diff
@@ -76,7 +76,7 @@ The new query search allows you to use complex queries in event monitors with ne
 
 ### Pipelines
 
-With pipelines, events are parsed and enriched by chaining them sequentially through processors. Processors extract meaningful information or attributes from semi-structured text to reuse as facets. Each event that comes through the pipelines is tested against every pipeline filter. If it matches a filter, then all the processors are applied sequentially before moving to the next pipeline.
+Datadog automatically parses JSON-formatted events. When events are not JSON-formatted, with pipelines, events are parsed and enriched by chaining them sequentially through processors. Processors extract meaningful information or attributes from semi-structured text to reuse as facets. Each event that comes through the pipelines is tested against every pipeline filter. If it matches a filter, then all the processors are applied sequentially before moving to the next pipeline.
```

Comment:
```suggestion
Datadog automatically parses JSON-formatted events. When events are not JSON-formatted, they are parsed and enriched by chaining them sequentially through a processing pipeline. Processors extract meaningful information or attributes from semi-structured text to reuse as facets. Each event that comes through the pipelines is tested against every pipeline filter. If it matches a filter, then all the processors are applied sequentially before moving to the next pipeline.
```
@Sayumi-n, the [logs document](https://docs.datadoghq.com/logs/log_configuration/pipelines/?tab=source#overview) says "...you can add value to your raw logs by sending them through a processing pipeline." Does the event pipeline happen automatically, or should we say something like this:
"You can add values to your events by sending them through a processing pipeline."